repository/fcm_token_repository.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from domain.entity.fcm_tokens import FCMToken
+from domain import session_scope, User
+import logging
+
+
+class FCMTokenRepository:
+    def __init__(self):
+        pass
+
+    def check_user_exists(self, user_id):
```

Comment:
Repositories should be conceptually separated. Each one should only contain functionality specific to their domain area. This is important to prevent code duplication, and assist future maintenance by allowing developers to have a clear mental model of the program as a whole. If you need to know where to put some function, just think about where you would look for it if it already existed.

This function should be in the user repository (it might already be?), when a future developer wants the same check, they'll probably look there to see if it exists. Please move it there. Then you can just call the function in the user repository the same way you did when calling functions in this repository from your API code.